spec/models/reports/repository_spec.rb

```diff
@@ -412,4 +419,61 @@ def refresh_views
       expect(repo.missed_visits_with_ltfu_rates[slug]).to eq(legacy_results[:missed_visits_with_ltfu_rate])
     end
   end
+
+  context "v2" do
```

Comment:
Are we replicating the v1 tests here? Can we instead put them in a loop so it's the exact same tests running the flag on/off? If v1 and v2 are functionally the same, we shouldn't need new tests no?

Reply:
Got it, thanks for the clarification!